customer-wallet/src/containers/Vendor/VendorWallet/styles.scss

```diff
@@ -1,47 +0,0 @@
-@import 'styles/variables.scss';
-
-.vendor-wallet {
-
-  .section-head {
-    .section-name {
-      position: relative;
-
-      .name {
-        font-weight: bold;
-      }
-
-      .avatar {
-        position: absolute;
-        top: 0;
-        right: 0;
-      }
-    }
-
-    .total-crypto {
-      margin-top: 20px;
-
-      .big {
-        font-weight: bold;
-        font-size: 20pt;
-      }
-    }
-  }
-
-  .section-content {
-
-    .reviews-list {
```

Comment:
We will find ourselves redoing a lot of styles work here. I think we made a bad choice. We should try our best to keep these styles. If we are deleting them, we need to make sure we 'port' them to new approach as we delete them, else we will be doing this head scratching css stuff again, and the lads who did this were good with their sass, way better than me for sure.

So I am thinking about whats the best way to save this design work. 

Of course we can go to git history and get them back, but it might be worth it to take a step back, roll back to a commit where these styles were in there and as I said port them over one by one. If we don't do that, we will be writing hand crafted css later, and none of us wants to do that, right?

Thoughts

Reply:
I agree, but Material 1.0 has new components and we could just show the reviews in a different way, like [Cards](https://material-ui.com/demos/cards/), or [Expansion Panels](https://material-ui.com/demos/expansion-panels/) or [Lists](https://material-ui.com/demos/lists/).

Or we can just resurrect these files as we go